src/ssr/index.umd.js

```diff
@@ -0,0 +1,2 @@
+export function extractStyles() {}
+export function autoHydrate() {}
```

Comment:
Should we throw a `console.warn` in here that these functions are not available in umd bundles for the browser?

Reply:
I think so. I will add soon.

Reply:
Added.~